app/controllers/posts_controller.rb

```diff
@@ -5,10 +5,14 @@ def index
   end
 
```

Comment:
**[OPTIONAL]**

- [ ] It would be nice if you make your README summary more detailed because it helps everyone involved in the project. By providing comprehensive information, it ensures that users and developers can easily understand the project's purpose, features, and how to contribute. It also saves time and effort for developers by serving as a reference guide for troubleshooting and configuration. Moreover, a well-documented README summary fosters transparency, and trust, and encourages people to try out and adopt the project. In short, it's all about making the project more accessible, collaborative, and user-friendly.

Reply:
what do you mean?

Reply:
I am talking about the readme summary. i am just suggesting you to make it better to make your readme descriptive and professional 👍